docs/src/main/sphinx/connector/kinesis.rst

```diff
@@ -33,22 +33,23 @@ Configuration properties
 
 The following configuration properties are available:
 
-======================================  =======================================================================
-Property Name                           Description
-======================================  =======================================================================
-``kinesis.access-key``                  Access key to AWS account or blank to use default provider chain
-``kinesis.secret-key``                  Secret key to AWS account or blank to use default provider chain
-``kinesis.aws-region``                  AWS region to be used to read kinesis stream from
-``kinesis.default-schema``              Default schema name for tables
-``kinesis.table-description-location``  Directory containing table description files
-``kinesis.hide-internal-columns``       Controls whether internal columns are part of the table schema or not
-``kinesis.batch-size``                  Maximum number of records to return in one batch
-``kinesis.fetch-attempts``              Read attempts made when no records returned and not caught up
-``kinesis.max-batches``                 Maximum batches to read from Kinesis in one single query
-``kinesis.sleep-time``                  Time for thread to sleep waiting to make next attempt to fetch batch
-``kinesis.iterator-from-timestamp``     Begin iterating from a given timestamp instead of the trim horizon
-``kinesis.iterator-offset-seconds``     Number of seconds before current time to start iterating
-======================================  =======================================================================
+======================================          =======================================================================
+Property Name                                   Description
+======================================          =======================================================================
```

Comment:
We need to extend left "=" to fix CI failure. This change should be extracted as a different commit.

Reply:
done

Reply:
We should add a space between property name and description. You can run `docs/build` command to validate docs syntax.

Reply:
its erroring out

Reply:
Below header should work. 
```
==============================================  =======================================================================
```

Reply:
ok,. committed. thanks.